notebooks/reinforcement_learning/solutions/contextual_bandits_with_tf_agents.ipynb

```diff
@@ -0,0 +1,1813 @@
+{
```

Comment:
<p>A description above about what each metric is will help understand what they are computing.</p><ul><li>RegretMetric - Computes the regret with respect to a baseline (ex: optimal_reward_fn)</li><li>SubOptimalArmsMetric - Computes the number of suboptimal arms with respect to a baseline (ex: optimal_action_fn)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='81'/>

Reply:
<p>Done!</p>